CSharpier.MsBuild does not support DotNet 8

Targets within the CSharpier.MsBuild project fail to run when the target framework is set to DotNet 8 while Dotnet SDK v7 is not installed on the host

How to replicate:

1. Ensure DotNet SDK v8 is the only SDK version available.
   ```
   winget uninstall Microsoft.DotNet.SDK.7
   winget install Microsoft.DotNet.SDK.8
   ```
1. Modify the file `CSharpier.MsBuild.Test.csproj`:
   - Change the `TargetFramework` to `net8.0`

1. Build the `CSharpier.MsBuild.Test` project with Release configuration
   ```
   cd Src/CSharpier.MsBuild.Test
   dotnet build -c Release
   ```

I see the following output:

```
MSBuild version 17.8.3+195e7f5a3 for .NET
  Determining projects to restore...
  All projects are up-to-date for restore.
  You must install or update .NET to run this application.

  App: C:\Users\adnarayan\.nuget\packages\csharpier.msbuild\0.26.0\tools\csharpier\net7.0\dotnet-csharpier.dll
  Architecture: x64
  Framework: 'Microsoft.NETCore.App', version '7.0.0' (x64)
  .NET location: C:\Program Files\dotnet\

  The following frameworks were found:
    3.1.32 at [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
    6.0.24 at [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
    8.0.0 at [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]

  Learn more:
  https://aka.ms/dotnet/app-launch-failed

  To install missing framework, download:
  https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=7.0.0&arch=x64&rid=win-x64&os=
  win10
C:\Users\adnarayan\.nuget\packages\csharpier.msbuild\0.26.0\build\CSharpier.MsBuild.targets(19,5): error MSB3073: The c
ommand "dotnet "C:\Users\adnarayan\.nuget\packages\csharpier.msbuild\0.26.0\build\../tools/csharpier/net7.0/dotnet-csha
rpier.dll"  --check --no-msbuild-check "C:\Users\adnarayan\source\csharpier\Src\CSharpier.MsBuild.Test" > NUL " exited
with code -2147450730. [C:\Users\adnarayan\source\csharpier\Src\CSharpier.MsBuild.Test\CSharpier.MsBuild.Test.csproj::T
argetFramework=net8.0]

Build FAILED.

C:\Users\adnarayan\.nuget\packages\csharpier.msbuild\0.26.0\build\CSharpier.MsBuild.targets(19,5): error MSB3073: The c
ommand "dotnet "C:\Users\adnarayan\.nuget\packages\csharpier.msbuild\0.26.0\build\../tools/csharpier/net7.0/dotnet-csha
rpier.dll"  --check --no-msbuild-check "C:\Users\adnarayan\source\csharpier\Src\CSharpier.MsBuild.Test" > NUL " exited
with code -2147450730. [C:\Users\adnarayan\source\csharpier\Src\CSharpier.MsBuild.Test\CSharpier.MsBuild.Test.csproj::T
argetFramework=net8.0]
    0 Warning(s)
    1 Error(s)

Time Elapsed 00:00:00.81
```

--- a/Docs/MSBuild.md
+++ b/Docs/MSBuild.md
@@ -51,7 +51,7 @@ Valid options are:
 - Debug
 
 ### Target Frameworks
-CSharpier.MSBuild will be run with net6.0 or net7.0 if the project targets one of the two frameworks. In cases where the project targets something else (net48, netstandard2.0) `CSharpier_FrameworkVersion` will default to net7.0
+CSharpier.MSBuild will be run with net6.0, net7.0 or net8.0 if the project targets one of the three frameworks. In cases where the project targets something else (net48, netstandard2.0) `CSharpier_FrameworkVersion` will default to net7.0
 This can be controlled with the following property. This property is required if the csproj is targeting < net6.0 (netstandard2.0, net48, etc) and net7.0 is not installed.
 ```xml
   <PropertyGroup>

--- a/Src/CSharpier.MsBuild/build/CSharpier.MsBuild.targets
+++ b/Src/CSharpier.MsBuild/build/CSharpier.MsBuild.targets
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup>
     <CSharpier_FrameworkVersion Condition="'$(CSharpier_FrameworkVersion)' == ''">$(TargetFramework)</CSharpier_FrameworkVersion>
-    <CSharpier_FrameworkVersion Condition="'$(CSharpier_FrameworkVersion)' != 'net6.0' and '$(CSharpier_FrameworkVersion)' != 'net7.0'">net7.0</CSharpier_FrameworkVersion>
+    <CSharpier_FrameworkVersion Condition="'$(CSharpier_FrameworkVersion)' != 'net6.0' and '$(CSharpier_FrameworkVersion)' != 'net7.0' and '$(CSharpier_FrameworkVersion)' != 'net8.0'">net7.0</CSharpier_FrameworkVersion>
     <CSharpierDllPath>$(MSBuildThisFileDirectory)../tools/csharpier/$(CSharpier_FrameworkVersion)/dotnet-csharpier.dll</CSharpierDllPath>
     <CSharpierArgs Condition="'$(CSharpier_Check)' == 'true'">$(CSharpierArgs) --check</CSharpierArgs>
     <CSharpierArgs Condition="'$(CSharpier_LogLevel)' != ''">$(CSharpierArgs) --loglevel $(CSharpier_LogLevel)</CSharpierArgs>

